src/app/AppLayout/AppLayout.tsx

```diff
@@ -336,7 +336,7 @@ export const AppLayout: React.FC<AppLayoutProps> = ({ children }) => {
           <ExternalLinkAltIcon color="grey" />
         </Icon>
       </ApplicationLauncherItem>,
-      <ApplicationLauncherItem key={'Guided tour'} onClick={handleOpenGuidedTour}>
+      <ApplicationLauncherItem key={'Guided Tour'} onClick={handleOpenGuidedTour}>
```

Comment:
Should this one be changed? The tour isn't a custom resource or some other entity, so it seems like it should not be capitalized as Tour.

Reply:
yes